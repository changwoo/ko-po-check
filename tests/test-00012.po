# -*-text-*-
# UI
#
msgid ""
msgstr ""
"Project-Id-Version: test 12\n"
"POT-Creation-Date: 2004-10-06 10:16+0900\n"
"PO-Revision-Date: 2004-10-06 10:16+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=EUC-KR\n"
"Content-Transfer-Encoding: 8bit\n"

#: test.c:1
msgid "<b>test</b>"
msgstr "<b테스트</b>"

#: test.c:2
msgid "<b>test</b>:"
msgstr "<b>테스트<b>:"

