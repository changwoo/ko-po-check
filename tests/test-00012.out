test-00012.po:17: <b>: 번역할 때 마크업을 똑같이 써야 합니다
test-00012.po:21: <b>: 번역할 때 마크업을 똑같이 써야 합니다
